### PR TITLE
Add wrapper: SEACR

### PR DESCRIPTION
**Pull request by stephanflemming**
FOR CONTRIBUTOR:
* [x] - I have read the [CONTRIBUTING.md](https://github.com/galaxyproject/tools-iuc/blob/master/CONTRIBUTING.md) document and this tool is appropriate for the tools-iuc repo.
* [x] - License permits unrestricted use (educational + commercial)
* [x] - This PR adds a new tool or tool collection
* [ ] - This PR updates an existing tool or tool collection
* [ ] - This PR does something else (explain below)

Github: https://github.com/FredHutch/SEACR
WebInterface: https://seacr.fredhutch.org/

(My) parameteroverview: https://docs.google.com/spreadsheets/d/1Y1OYbLSiyXT73HNCPwxeNtgupiZmyWyOFfb9wUg5WvI/edit?usp=sharing

Fix https://github.com/galaxyproject/tools-iuc/issues/2614

### CODE REVIEW

**Comment by stephanflemming** [comment]
Update bioconda recipe https://github.com/bioconda/bioconda-recipes/pull/20154

**Comment by stephanflemming** [comment]
I don't have testdata for using the control bedgraph parameter. Beside that, the tool should be complete and can be reviewed.

File: tools/seacr/seacr.xml [hunk 1]
```diff
@@ -0,0 +1,131 @@
...
+        ]]></command>
+    <inputs>
+        <param argument="bedgraph" type="data" format="bedgraph" multiple="false" label="Select target bedgraph file" help="Target data bedgraph file in UCSC bedgraph format that omits regions containing 0 signal."/>
+        <conditional name="control_cond">
+            <param name="control_sel" type="select" multiple="false" label="Select control type">
+                <option value="f">Control bedgraph</option>
+                <option value="t">Threshold</option>
+            </param>
+            <when value="f">
+                <param argument="f" type="data" format="bedgraph" multiple="false" label="Select control bedgraph file" help="Control (IgG) data bedgraph file to generate an empirical threshold for peak calling."/>
```
Comment (bgruening) [inline tools/seacr/seacr.xml]: multiple=false is the default and does not need to be specifid

**Comment by bgruening** [comment]
Danke.

**Comment by bgruening** [comment]
Danke.

**Pull request merged**